frontend/apps/web/src/components/system/Overview.tsx

```diff
@@ -1,20 +1,66 @@
-import React from 'react'
-import { CardLayout, ProgressBar, StackLayout, Text, Table } from '@filecoin/ui'
-import styled from 'styled-components'
-import { System } from '@filecoin/types'
-
 import { getButton } from '@/pages/tests'
-
+import { filterItems } from '@filecoin/core'
+import { System } from '@filecoin/types'
+import {
+  CardLayout,
+  Dropdown,
+  ProgressBar,
+  SearchInput,
+  StackLayout,
+  Table,
+  Text,
+} from '@filecoin/ui'
+import React, { useState } from 'react'
+import styled from 'styled-components'
 interface Props {
   system: System
 }
 
 export const Overview: React.FC<Props> = ({ system }) => {
   const totalSubsystems = system.subsystems.length
+  const [searchTerm, setSearchTerm] = useState('')
+  const [selectedFilter, setSelectedFilter] = useState('')
+
+  const calculateResults = () => {
+    if (searchTerm) {
+      const searchTermResult = filterItems(
```

Comment:
Point of this is for example user clicks on filter by bad score.
And now we have a list of all bad scores, But still, a user could also search those bad scores.
So note for you, that we need to switch this. First we do a filter if there is any, then we do a search, and finally do a pagination.